src/cli/cli.cpp

```diff
@@ -557,8 +562,10 @@ void Interpreter::ProcessPanId(int argc, char *argv[])
 void Interpreter::HandleEchoResponse(void *aContext, Message &aMessage, const Ip6::MessageInfo &aMessageInfo)
 {
     Ip6::IcmpHeader icmp6Header;
+    uint32_t timestamp;
 
     aMessage.Read(aMessage.GetOffset(), sizeof(icmp6Header), &icmp6Header);
+    aMessage.Read(aMessage.GetOffset() + sizeof(icmp6Header), sizeof(uint32_t), &timestamp);
```

Comment:
Make sure to check the return value here, in case the message length is too small to hold the timestamp value.